apps/i18n/common/en_us.json

```diff
@@ -61,14 +61,12 @@
   "censusFrequency1to3": "1-3 hours per week",
   "censusFrequency3plus": "3+ hours per week",
   "censusHeading": "Tell us about your school",
-  "censusHocAll": "All students do an Hour of Code",
-  "censusHocSome": "Some students do an Hour of Code",
   "censusHowMuch": "How much coding/computer programming is taught at this school? (assume for the purposes of this question that this does not include HTML/CSS, Web design, or how to use apps)",
-  "census10HourAll": "All students take at least 10 hours of computer programming integrated into a non-Computer Science course such as Art, Math, or Science",
-  "census10HourSome": "Some students take at least 10 hours of computer programming integrated into a non-Computer Science course such as Art, Math, or Science",
-  "census20HourAll":"All students take a semester or year-long computer science course that includes at least 20 hours of coding/programming",
-  "census20HourSome": "Some students take a semester or year-long computer science course that includes at least 20 hours of coding/programming",
   "censusGameDesign": "Game design using game layout tools without coding or computer programming",
+  "censusHowManyAfterSchool": "How many students do computer programming in an after-school program?",
+  "censusHowManyHoC": "How many students do an Hour of Code?",
+  "censusHowManyTenHours": "How many students take at least 10 hours of computer programming integrated into a non-Computer Science course (such as TechEd, Math, Science, Art, Library or general classroom/homeroom)?",
+  "censusHowManyTwentyHours": "How many students take a semester or year-long computer science course that includes at least 20 hours of coding/computer programming?",
```

Comment:
possibly: "Semester- or year-long" but that's very nitty and I'm not sure we'd do that :)